### PR TITLE
Rebuild path tile build/remove dragging in single tile mode

### PR DESCRIPTION
**Pull request by Alberth289346**
Branch tag doesn't really describe what it does :)

### CODE REVIEW

File: src/path_gui.cpp [hunk 1]
```diff
@@ -228,6 +229,25 @@ PathBuildGui::~PathBuildGui()
...
+{
+	if ((m_state & (MB_LEFT | MB_RIGHT)) == 0) return; // No buttons pressed.
+
+	const Voxel *v = _world.GetVoxel(this->mouse_pos);
+	if (!v) return; // No voxel -> no ground there.
+
+	uint16 instance = v->GetInstance();
+	if (instance == SRI_FREE && (m_state & MB_LEFT) != 0) {
+		this->BuildSinglePath(this->mouse_pos);
+	} else if (instance == SRI_PATH && (m_state & MB_RIGHT) != 0) {
```
Comment (ghost) [inline src/path_gui.cpp:246]: We will probably also want to allow for changing the current path type with [ChangePathTile](https://github.com/FreeRCT/FreeRCT/blob/master/src/path_build.cpp#L288). -> [Documentation](http://freerct.github.io/FreeRCT/path__build_8cpp.html#ad6b3116dd283d5d399d8998df4eff0f0)

**Comment by Alberth289346** [comment]
Rebuild path tiles has been added

File: src/path_gui.cpp [hunk 2]
```diff
@@ -228,6 +229,25 @@ PathBuildGui::~PathBuildGui()
 	this->SetSelector(nullptr);
 }
 
+/**
+ * Try to add (if LMB pressed) or remove (if RMB pressed) a path tile at the vpxel pointed to by the mouse.
```
Comment (ghost) [inline src/path_gui.cpp]: Small typo: 'vpxel' -> 'voxel'.

**Comment by ghost** [comment]
Awesome! Looks good to me.

**Comment by Alberth289346** [comment]
Fixed

**Pull request merged**